### PR TITLE
Spur trimming update

### PR DESCRIPTION
**Pull request by pb-jchin**

### CODE REVIEW

File: falcon_kit/mains/ovlp_to_graph.py [hunk 1]
```diff
@@ -1146,55 +1197,75 @@ def main(argv=sys.argv):
...
-                rs = reverse_end(t)
-                rt = reverse_end(s)
-                rv = reverse_end(v)
-                edges_to_remove.add( (rs, rt, rv) )
-                length, score, edges, type_ = u_edge_data[ (rs, rt, rv) ]
-                u_edge_data[ (rs, rt, rv) ] = length, score, edges, "spur:2"
 
-    for s, t, v in list(edges_to_remove):
-        ug2.remove_edge( s, t, key= v)
+            s_path = nx.shortest_path( ug2, n, b_node )
```
Comment (pb-cdunn) [inline falcon_kit/mains/ovlp_to_graph.py]: This line is unreachable. If `with_extern_node` is false, then we `continue` at 1231; if it's true, then we `break` at 1228.

Comment (pb-jchin) [inline falcon_kit/mains/ovlp_to_graph.py]: The `with_extern_node` will be true when `if v not in n_egg_node_set`. Once the variable is set to `True` the rest are execute.

```
1221             for v in  b_in_nodes:
1222                 if v not in n_egg_node_set:
1223                     with_extern_node = True
1224                     break
```

BTW, there is a typo `n_egg_node_set` should be `n_ego_node_set`

Comment (pb-cdunn) [inline falcon_kit/mains/ovlp_to_graph.py]: Ok. I see it now. I nice way to write this in Python is
```python
for x in outer_loop:
  for v in b_in_nodes:
    if blah:
      break
  else:
    continue
```
That does exactly what you want.

Comment (pb-jchin) [inline falcon_kit/mains/ovlp_to_graph.py]: but as you pointed out before, the `continue` statement is redundant

File: falcon_kit/mains/ovlp_to_graph.py [hunk 2]
```diff
@@ -1146,55 +1197,75 @@ def main(argv=sys.argv):
...
+            total_length = 0
+            for v2 in s_path[1:]:
+                for s, t, v in ug2.out_edges(v1, keys=True):
+                    if t != v2:
+                       continue
+                    length, score, edges, type_ = u_edge_data[ (s, t, v) ]
+                    total_length += length
+                v1 = v2
+
+            if total_length >= 50000:
```
Comment (pb-cdunn) [inline falcon_kit/mains/ovlp_to_graph.py]: Should we break out of those 2 inner for-loops early when we exceed 50000? That would be much messier code, but could save time in extreme cases. Worth it?

**Pull request merged**